tests/certificate/certificate.helper.js

```diff
@@ -89,6 +89,37 @@ const getCertificateById = async (id, operations) => {
   return result.data.getCertificateById;
 };
 
+const getCertificateByName = async (name, operations) => {
+  const result = await operations.query({
+    query: gql`
+      query ($name: String!) {
+        getCertificateByName(name: $name) {
+          ... on Certificate {
+            _id
+            name
+            isExpired
+            isUsed
+            value
+            email
+            ownedBy {
+              _id
```

Comment:
Are you sure we'll store the _id property inside ownedBy object?

Reply:
The _id property is already stored in it, as the `ownedBy` field has a graphql type of `User`.
I'd rather ask such a question: how will the ownedBy property be used when testing? If it won't, then maybe it's redundant.